src/main/java/org/reactome/orthoinference/EventsInferrer.java

```diff
@@ -80,13 +73,17 @@ public static void inferEvents(Properties props, String species) throws Exceptio
 		setDbAdaptors(dbAdaptor);
 
 		releaseVersion = props.getProperty("releaseNumber");
-		String pathToOrthopairs = Paths.get(props.getProperty("pathToOrthopairs") + releaseVersion).toString();
+		String pathToOrthopairs = props.getProperty("pathToOrthopairs");
 		String pathToSpeciesConfig = props.getProperty("pathToSpeciesConfig");
 		String dateOfRelease = props.getProperty("dateOfRelease");
 		int personId = Integer.valueOf(props.getProperty("personId"));
 		setReleaseDates(dateOfRelease);
 
 		String pathToSkipList = props.getProperty("pathToOrthoinferenceSkipList");
+		if (!Files.exists(Paths.get(pathToSkipList))) {
+			logger.fatal("Unable to locate skiplist file: " + pathToSkipList);
+			return;
+		}
```

Comment:
If this condition is fatal, should an exception be thrown rather than returning from the method?  Or if returning is okay, maybe `logger.error` is more appropriate?

Reply:
I agree: using `return;` to break out of a void method doesn't feel very clean. There are exceptions that can be thrown that would also break the execution, or you could use `System.exit(-1)` to indicate the program has exited with an error.

Reply:
OK sure, I was just mimicking how it was already done in this code. I'll replace all examples of 'return' in this code, meant to terminate the program, with the `System.exit(-1)`

Reply:
OK all fatal 'returns' in the code have been changed to `System.exit(-1)`